ratings/tests.py

```diff
@@ -54,10 +54,11 @@ def test_rating_change(self):
         """ Change your existing rating. """
         loggedin = self.client.login(username="testuser1", password="testpass")
         self.assertTrue(loggedin)
+
         r = ratings.models.Rating.objects.create(object_id=self.sound.id, content_type=self.ct, user_id=self.user1.id, rating=4)
 
         resp = self.client.get("/ratings/add/%s/%s/%s/" % (self.ct.id, self.sound.id, 5))
-        newr = ratings.models.Rating.objects.get(id=r.id)
+        newr = ratings.models.Rating.objects.first()
```

Comment:
this is OK because in the next line we check that we only have 1 `Rating` object, but I preferred the original line because we explicitly check that the ratings object is the same based on its ID.